pkg/types/azure/platform.go

```diff
@@ -11,6 +11,9 @@ type Platform struct {
 	// BaseDomainResourceGroupName specifies the resource group where the Azure DNS zone for the base domain is found.
 	BaseDomainResourceGroupName string `json:"baseDomainResourceGroupName,omitempty"`
 
+	// PublicZoneID specifies the ID for the public DNS Zone
+	PublicZoneID string `json:"publicZoneID"`
```

Comment:
Do you need additional field to specify the zone in Azure.. ??
https://github.com/openshift/installer/blob/7ed7015edb5a51293c0a2266f0770a324e5d7462/pkg/types/azure/platform.go#L11-L12

that already restricts the zone to single resource, ie. (basedomain resourcegroup, basedomain) => only one dns zone right?

Why do we need this?

Reply:
You are correct. Only the domain and resource group are necessary to uniquely identify the zone. I have removed this from the PR.